pkg/opcfg/config.go

```diff
@@ -375,3 +375,18 @@ func getStackTrace() zap.Option {
 	}
 	return zap.AddStacktrace(zapcore.LevelEnabler(lvl))
 }
+
+// GetVdbMaxBackoffDuration returns maximum backoff requeue duration in milliseconds for vdb controller
+func GetVdbMaxBackoffDuration() int {
+	const envName = "VDB_MAX_BACKOFF_DURATION"
+	duration := lookupStringEnvVar(envName, envCanNotExist)
+	if duration == "" {
+		// Default value is 1000ms
+		return 1000
+	}
+	valInt, err := strconv.Atoi(duration)
+	if err != nil || valInt < 0 {
+		dieIfNotValid(envName)
```

Comment:
You should return the default value in this case too. see lookupIntEnvVar. Also use a constant for the default value.

Reply:
`lookupIntEnvVar` doesn't return any value when err != nil (dieIfNotValid() will exit the process).  I think if the user provided the wrong value, the operator should exit and tell them their value isn't acceptable. We shouldn't replace their wrong value with a default value.

If the default value occurred more than once in the code, I will use a constant value, but it didn't so I just use a literal constant.